src/fmu/tools/sensitivities/create_design.py

```diff
@@ -194,6 +196,18 @@ def to_xlsx(
         defaults.to_excel(
             xlsxwriter, sheet_name=defaultsheet, index=False, header=False
         )
+
+        version_info = pd.DataFrame(
+            {
+                "Description": ["Created using fmu-tools version:", "Created on:"],
+                "Value": [
+                    fmu.tools.__version__,
+                    datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
```

Comment:
Recommend to use `datetime.now().isoformat(sep=" ", timespec="seconds")`

Reply:
Thanks, fixed.